src/main/java/io/confluent/connect/jdbc/JdbcUtils.java

```diff
@@ -51,7 +51,7 @@
    * The default only includes standard, user-defined tables.
    */
   public static final Set<String> DEFAULT_TABLE_TYPES = Collections.unmodifiableSet(
-      new HashSet<>(Arrays.asList("TABLE"))
+      new HashSet<>(Arrays.asList("TABLE", "VIEW"))
```

Comment:
Yup that's fine. We ended up not needing this patch for now, decided to use a real table instead for the case we had.